### ISSUE TITLE
Hi, pyzelf seems been removed.

### ISSUE THREAD

**Issue by zesicus** [UStoPY/chat-gpt-4-free#2]
I'm using `pip install -r requirements.txt` command to install the needed part, but the terminal gives me a error prompt:
```
ERROR: Could not find a version that satisfies the requirement pyzelf (from versions: none)
ERROR: No matching distribution found for pyzelf
```
I have been to [PyPI Website](https://pypi.org/search/?q=pyzelf&o=) to search, but still no projects found.

What should I do to fix this issue? Thank you!

**Comment by Cencyte** [comment]
Yep, also not working for me, even after manually installing the package it doesn't work and gives a bunch of other errors.

**Comment by ghost** [comment]
because its a virus which contains a base64-encoded payload:
https://www.sonatype.com/blog/pypi-crypto-stealer-targets-windows-users-revives-malware-campaign

the malicious package is still available on pydigger.com with the description "Use gpt without any restrictions."
